### PR TITLE
C2-11081 Let the user setting his zoom value from the General Settings menu

### PR DESCRIPTION
**Pull request by BachLeFPT**
## Description
Let the user setting his zoom value from the General Settings menu

## JIRA ticket

[C2-11081](https://perzoinc.atlassian.net/browse/C2-11081)

## Related PRs
https://github.com/SymphonyOSF/SFE-Lite/pull/7519

## Demo
**- Old SDA**
C2-11081_old_sda

**- Browser**
C2-11081_browser

- New SDA
https://user-images.githubusercontent.com/52144470/123741242-e1a02500-d8d3-11eb-82e0-2b4df0c98bc3.mp4

### CODE REVIEW

**Comment by BachLeFPT** [comment]
@sbenmoussati Please help me check this PR.
I haven't permission to add reviewer or any label...
Thanks!

File: src/renderer/ssf-api.ts [hunk 1]
```diff
@@ -674,6 +681,30 @@ export class SSFApi {
       notificationId,
     });
   }
+
+  /**
+   * Get zoom level
+   *
+   * @param {string} locale - language identifier and a region identifier
```
Comment (sbenmoussati) [inline src/renderer/ssf-api.ts:688]: Please remove @param and @example as they don't correspond to this method

File: src/renderer/ssf-api.ts [hunk 2]
```diff
@@ -674,6 +681,30 @@ export class SSFApi {
...
+  public getZoomLevel(): Promise<any> {
+    return new Promise((resolve) => {
+      resolve(webFrame.getZoomFactor());
+    });
+  }
+
+  /**
+   * Sets zoom level
+   *
+   * @param {string} locale - language identifier and a region identifier
```
Comment (sbenmoussati) [inline src/renderer/ssf-api.ts:700]: same here for @param and @example

File: src/renderer/ssf-api.ts [hunk 3]
```diff
@@ -147,6 +147,13 @@ const throttledRestart = throttle(() => {
   });
 }, 1000);
 
+const throttledSetZoomLevel = throttle((zoomLevel) => {
+  local.ipcRenderer.send(apiName.symphonyApi, {
+    cmd: apiCmds.setZoomLevel,
+    zoomLevel,
+  });
+}, 1000);
```
Comment (sbenmoussati) [inline src/renderer/ssf-api.ts:155]: Would it be possible to replace all these delays (1000) by a const please?
Something like ```const DEFAULT_THROTTLE = 1000;```

**Review by sbenmoussati — CHANGES_REQUESTED** [review]

**Review by mattias-symphony — APPROVED** [review]

File: src/renderer/ssf-api.ts (hunk 1 above)
Comment (BachLeFPT) [reply src/renderer/ssf-api.ts]: @sbenmoussati 
I updated it.
Thanks!

File: src/renderer/ssf-api.ts (hunk 2 above)
Comment (BachLeFPT) [reply src/renderer/ssf-api.ts]: @sbenmoussati 
I updated it.
Thanks!

File: src/renderer/ssf-api.ts (hunk 3 above)
Comment (BachLeFPT) [reply src/renderer/ssf-api.ts]: @sbenmoussati 
I updated it.
Thanks!

**Review by sbenmoussati — APPROVED** [review]

**Pull request merged**